app/components/gcse_row_content_component.html.erb

```diff
@@ -0,0 +1,31 @@
+<% if course.gcse_section_complete? %>
+  <p class="govuk-summary-list-value govuk-!-margin-top-0">
+    <%= required_gcse_content(course) %>
+  </p>
+
+  <p class="govuk-summary-list-value govuk-!-margin-top-0">
+    <%= pending_gcse_content(course) %>
+  </p>
+
+  <p class="govuk-summary-list-value govuk-!-margin-top-0">
+    <%= gcse_equivalency_content(course) %>
+  </p>
+
+  <% if course.additional_gcse_equivalencies %>
+    <p class="govuk-summary-list-value">
+      <%= course.additional_gcse_equivalencies %>
+    </p>
```

Comment:
I think the markup is slightly wrong here. If I’m right, this content already appears within a `govuk-summary-list-value` container, so just need to add paragraphs:

```suggestion
  <p class="govuk-body">
    <%= required_gcse_content(course) %>
  </p>

  <p class="govuk-body">
    <%= pending_gcse_content(course) %>
  </p>

  <p class="govuk-body">
    <%= gcse_equivalency_content(course) %>
  </p>

  <% if course.additional_gcse_equivalencies %>
    <p class="govuk-body">
      <%= course.additional_gcse_equivalencies %>
    </p>
```